igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/JenkinsBuildMonitor.groovy

```diff
@@ -95,14 +99,14 @@ class JenkinsBuildMonitor implements PollingMonitor {
     boolean isInService() {
         if (discoveryClient == null) {
             log.info("no DiscoveryClient, assuming InService")
-            true
+            pollingEnabled
```

Comment:
for readability this should be `return pollingEnabled` since the return value is happening in the middle of the method